docsource/docs/quickstart/tutorial.md

```diff
@@ -0,0 +1,127 @@
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+*   About 30 mins of time.
+*   A Google Account, for use on the Google Cloud Platform.
+*   Working instance of CRMint, if not please read [how to deploy CRMint on GCP](quickstart/index.html).
+
+### Costs
+
+This tutorial uses billable components of Cloud Platform, including:
+
+* Google BigQuery
+* BigQuery ML
+
+You incur charges for:
+
+* [Storing](https://cloud.google.com/bigquery/pricing#active_storage) your ML model and training data in BigQuery
+    * The first 10 GB of storage is free each month.
+* [Querying](https://cloud.google.com/bigquery/pricing#on_demand_pricing) data in BigQuery
+    * The first 1 TB is free each month.
+    * If you are using flat-rate pricing, query costs are included in the monthly flat-rate price.
+* Running [BigQuery ML SQL statements](https://cloud.google.com/bigquery/pricing#bqml)
+
+## Let's predict the price of houses
+
+As usually in Data-Science, let's try to predict the price of real-estate. We do have some open-data for Brasil thanks to [Properati](http://properati.com).
+
+Here is the simple model we will use to achieve our modest goal:
+
+```sql
+#standardSQL
+
+CREATE MODEL `predict_realestate_brasil.price_model`
+OPTIONS (
+    model_type='linear_reg',
+    early_stop=False,
+    max_iterations=20,
+    input_label_cols=['price']
+) AS
+SELECT
+  property_type,
+  state_name,
+  price,
+  surface_covered_in_m2,
+  rooms,
+  (surface_covered_in_m2 / rooms) AS room_avg_surface,
+  LENGTH(description) AS desc_length
+FROM `properati-data-public.properties_br.properties_sell_201802`
+WHERE
+  currency = 'BRL'
+  AND surface_covered_in_m2 IS NOT NULL
+  AND surface_covered_in_m2 > 1
+  AND rooms IS NOT NULL
+  AND MOD(ABS(FARM_FINGERPRINT(id)), 10) < 7
+```
+
+## Create a dataset in BigQuery
+
+1.  Indicate the name of your project
+
+    <input id="project-id" placeholder="Insert Project ID here, e.g. flying-tiger-112301" data-target-id="gcp-console">
+
+1.  Open your <a id="gcp-console" href="https://console.cloud.google.com/bigquery?project=placeholder" target="_blank">BigQuery console</a>
+
+1.  Select your project name on the left sidebar.
+
+1.  Create a dataset with the button on the right &ldquo;Create Dataset&rdquo;.
+
+    ![Create Dataset button](../../img/gcp-bq-createdataset-button.png)
+
+1.  Name your dataset `predict_realestate_brasil`, and choose to store the data in the `US` location.
+
+    ![Dataset configuration](../../img/gcp-bq-createdataset-conf.png)
+
+## Create the training pipeline in CRMint
+
+1.  Open your CRMint instance <a href="https://xxxxxx.appspot.com" id="crm-launch" target="_blank">&lt;Project ID not set&gt;</a>.
+
+1.  Download the pre-built pipeline [train_evaluate_model.json](https://storage.googleapis.com/crmint-public/templates/pipelines/bqml/train_evaluate_model.json).
```

Comment:
I believe this attribute only works for assets from the same domains. Because [this](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_html_download_link) works with an image from a relative path, but it doesn't work when I use a link from another domain :/

Reply:
Here are some clues: https://stackoverflow.com/q/49474775